CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 #### Bugfixes
 
+- Fixed infinite repetition issue on mv with wildcard same for source and destination ([#168](https://github.com/peak/s5cmd/issues/168))
```

Comment:
We also have the same issue for **cp**. If we copy/move S3 objects into sub prefixes, this issue may occur.

Reply:
since mv uses cp operations and cp depend on listing objects, doesn't this solve other related issues as well? I will check that.